src/storage/mod.rs

```diff
@@ -119,6 +119,14 @@
 //!     ) -> Result<()> {
 //!         unimplemented!()
 //!     }
+//!
+//!     async fn md5<P: AsRef<Path> + Send + Debug>(
```

Comment:
If we have a default implementation then perhaps we should omit this here for simplicity sake.

Reply:
yes